website/content/docs/sync/gcpsm.mdx

```diff
@@ -264,6 +264,37 @@ This overrides the project ID derived from the service account JSON credentials
 default credentials. The service account must be [authorized](https://cloud.google.com/iam/docs/service-account-overview#locations)
 to perform Secret Manager actions in the target project.
 
+## Security
+
+Access to secrets can be allowed or restricted through the use of their
+[resource names by using conditions](https://cloud.google.com/iam/docs/conditions-resource-attributes#resource-name). For secrets,
+the fully qualified resource name is required in the format of `projects/<project_number>/secrets/<secret_name>` Take note that
+this requires the project _number_, not the project _ID_.
```

Comment:
Any reason you did not use the tag/label condition like for AWS?

Tags are more commonly used as conditions and they do not depend on user-defined fields like the name template making them less error prone. We should guide users to the solution with less pitfalls.

Reply:
From my research, GCP doesn't seem to support resource tags on individual secrets, and additionally it also doesn't support using secret labels as IAM conditions.

I tried finding alternatives but the only solution I was able to find was creating this rule via secret name. Are you already familiar with a better option? I did find feature requests from users for Google to add tags to secrets but in the meantime I don't believe there's other options.

Reply:
My bad you're right, [GCP SM only has labels and labels cannot be used for IAM conditions](https://cloud.google.com/compute/docs/labeling-resources#labels_and_tags). I thought they'd have the same capabilities.